website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+---
+title: '0.72 - Beta Support for Package Exports and Symlinks'
```

Comment:
I think we could create a more exciting title. After my comments are addressed, we could try to see if we can get chatGPT to generate some cool titles ahah

Reply:
I think the highlight features should be package exports and symlinks. What are you thinking?

Reply:
here's 10 variations that chatGPT (3.5) offered me:
1. "Introducing React Native 0.72: Enhanced Metro Features and Developer Experience Improvements"
2. "React Native 0.72 Release: Metro Upgrades, Developer Experience Enhancements, and More"
3. "What's New in React Native 0.72: Improved Metro and Exciting Developer Experience Updates"
4. "Discover the Latest React Native Version 0.72: Enhanced Metro, Developer Experience, and ANR Mitigation"
5. "Breaking News: React Native 0.72 Unveiled with Metro Enhancements and Developer Experience Improvements"
6. "React Native 0.72 Now Available: Unveiling Symlink Support, Package Exports, and More"
7. "Unlocking New Possibilities: React Native 0.72 Release Introduces Metro Enhancements and Developer Improvements"
8. "React Native 0.72: Symlink Support, Package Exports, and Noteworthy Developer Experience Upgrades"
9. "Level Up Your React Native Apps with Version 0.72: Metro Enhancements and Developer Experience Boosts"
10. "What's Inside React Native 0.72: Empowering Developers with Metro Updates and Enhanced Experiences"